src/routes/api/auth/index.ts

```diff
@@ -2,14 +2,17 @@ import {
   FastifyPluginAsyncTypebox,
   Type
 } from '@fastify/type-provider-typebox'
-import { CredentialsSchema, Credentials } from '../../../schemas/auth.js'
+import { Credentials } from '../../../schemas/auth.js'
 
 const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
   fastify.post(
     '/login',
     {
       schema: {
-        body: CredentialsSchema,
+        body: Type.Object({
+          email: Type.String(),
```

Comment:
This is not good, we must process only valid payload. Use updated `CredentialsSchema` with email format constraint.

Reply:
(same comment than before)